app/components/Settings/Tabs/Tab.react.js

```diff
@@ -38,7 +52,7 @@ export default class Tab extends Component {
 
                 <p className={styles.tabIdentifier}>
                     <span>
-                        {username}@{host}
+                        {label}
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1280389/21062154/743857b0-be1d-11e6-9f0f-5c6b1f574b97.png)

@cldougl - here's a fix to update the tab names. they'll match the same ones in the dropdown in `/create`

![image](https://cloud.githubusercontent.com/assets/1280389/21062183/a8227268-be1d-11e6-9b3e-056d0b956909.png)